src/Tribe/Properties/Event_Data.php

```diff
@@ -34,7 +34,7 @@ public function get_event_values( $prefix, $post_id ) {
 			],
 			[
 				'property' => $prefix . 'event_organizer',
```

Comment:
We are sending the names to Hubspot using this code and the IDs are not required.